test/gateway/eip20_cogateway/helpers/helper.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const web3 = require('../../../test_lib/web3.js');
+
+
+const EIP20CoGatewayHelper = function() {
+
+};
+
+EIP20CoGatewayHelper.prototype = {
+    
+    /**
+     * It sets the cogateway address.
+     */
+    setGateway: async function(coGateway) {
+    
+        this.coGateway = coGateway;
+    
+    },
+    
+    /**
+     * Generate the stake type hash. This is as per EIP-712
+     *
+     * @return {string} message type hash.
+     */
+    redeemTypeHash: async function() {
+        return web3.utils.soliditySha3(
+            web3.eth.abi.encodeParameter(
```

Comment:
`web3.utils.soliditySha3` already does packed abi encoding. Is it required to encode it manually before?

https://web3js.readthedocs.io/en/1.0/web3-utils.html#soliditysha3